src/Symfony/Component/Validator/Constraints/Video.php

```diff
@@ -0,0 +1,196 @@
+        ?string $maxPixelsMessage = null,
+        ?string $maxRatioMessage = null,
+        ?string $minRatioMessage = null,
+        ?string $allowSquareMessage = null,
+        ?string $allowLandscapeMessage = null,
+        ?string $allowPortraitMessage = null,
+        ?string $corruptedMessage = null,
+        ?array $groups = null,
+        mixed $payload = null,
+    ) {
+        parent::__construct(
+            $options,
+            $maxSize,
+            $binaryFormat,
+            $mimeTypes,
+            $filenameMaxLength,
+            $notFoundMessage,
+            $notReadableMessage,
+            $maxSizeMessage,
+            $mimeTypesMessage,
+            $disallowEmptyMessage,
+            $filenameTooLongMessage,
+            $uploadIniSizeErrorMessage,
+            $uploadFormSizeErrorMessage,
+            $uploadPartialErrorMessage,
+            $uploadNoFileErrorMessage,
+            $uploadNoTmpDirErrorMessage,
+            $uploadCantWriteErrorMessage,
+            $uploadExtensionErrorMessage,
+            $uploadErrorMessage,
+            $groups,
+            $payload
+        );
+
+        $this->minWidth = $minWidth ?? $this->minWidth;
+        $this->maxWidth = $maxWidth ?? $this->maxWidth;
+        $this->maxHeight = $maxHeight ?? $this->maxHeight;
+        $this->minHeight = $minHeight ?? $this->minHeight;
+        $this->maxRatio = $maxRatio ?? $this->maxRatio;
+        $this->minRatio = $minRatio ?? $this->minRatio;
+        $this->minPixels = $minPixels ?? $this->minPixels;
+        $this->maxPixels = $maxPixels ?? $this->maxPixels;
+        $this->allowSquare = $allowSquare ?? $this->allowSquare;
+        $this->allowLandscape = $allowLandscape ?? $this->allowLandscape;
+        $this->allowPortrait = $allowPortrait ?? $this->allowPortrait;
+        $this->sizeNotDetectedMessage = $sizeNotDetectedMessage ?? $this->sizeNotDetectedMessage;
+        $this->maxWidthMessage = $maxWidthMessage ?? $this->maxWidthMessage;
+        $this->minWidthMessage = $minWidthMessage ?? $this->minWidthMessage;
+        $this->maxHeightMessage = $maxHeightMessage ?? $this->maxHeightMessage;
+        $this->minHeightMessage = $minHeightMessage ?? $this->minHeightMessage;
+        $this->minPixelsMessage = $minPixelsMessage ?? $this->minPixelsMessage;
+        $this->maxPixelsMessage = $maxPixelsMessage ?? $this->maxPixelsMessage;
+        $this->maxRatioMessage = $maxRatioMessage ?? $this->maxRatioMessage;
+        $this->minRatioMessage = $minRatioMessage ?? $this->minRatioMessage;
+        $this->allowSquareMessage = $allowSquareMessage ?? $this->allowSquareMessage;
+        $this->allowLandscapeMessage = $allowLandscapeMessage ?? $this->allowLandscapeMessage;
+        $this->allowPortraitMessage = $allowPortraitMessage ?? $this->allowPortraitMessage;
+        $this->corruptedMessage = $corruptedMessage ?? $this->corruptedMessage;
+
+        if (!class_exists(\FFMpeg\FFProbe::class)) {
```

Comment:
let's do this check at the very start of the method

Reply:
fixed